### PR TITLE
Markodonnell deriv patch 3

### PR DESCRIPTION
**Pull request by markodonnell-deriv**
Further work on template plus example smoke tests

### CODE REVIEW

File: cypress/e2e/smoke/aboutUs.cy.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import '@testing-library/cypress/add-commands'
+
+describe('About us page', () => {
+
+  beforeEach(() => {
```
Comment (habib-deriv) [inline cypress/e2e/smoke/aboutUs.cy.js:5]: We can remove the `beforeEach` if no longer needed
```suggestion
  beforeEach(() => {
```

File: cypress/e2e/smoke/legal.cy.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import '@testing-library/cypress/add-commands'
+
+describe('Legal page', () => {
+
+  beforeEach(() => {
```
Comment (habib-deriv) [inline cypress/e2e/smoke/legal.cy.js:5]: We can remove the `beforeEach` if no longer needed
```suggestion
  beforeEach(() => {
```

File: cypress/e2e/smoke/markets.cy.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import '@testing-library/cypress/add-commands'
+
+describe('Markets page', () => {
+
+  beforeEach(() => {
```
Comment (habib-deriv) [inline cypress/e2e/smoke/markets.cy.js:5]: We can remove the `beforeEach` if no longer needed
```suggestion
```

File: cypress/e2e/smoke/partners.cy.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import '@testing-library/cypress/add-commands'
+
+describe('Partners page', () => {
+
+  beforeEach(() => {
```
Comment (habib-deriv) [inline cypress/e2e/smoke/partners.cy.js:5]: We can remove the `beforeEach` if no longer needed
```suggestion
```

File: cypress/e2e/smoke/resources.cy.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import '@testing-library/cypress/add-commands'
+
+describe('Resources page', () => {
+
+  beforeEach(() => {
```
Comment (habib-deriv) [inline cypress/e2e/smoke/resources.cy.js:5]: We can remove the `beforeEach` if no longer needed
```suggestion
```

File: cypress/e2e/smoke/trade.cy.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
+import '@testing-library/cypress/add-commands'
+
+describe('Trade page', () => {
+
+  beforeEach(() => {
```
Comment (habib-deriv) [inline cypress/e2e/smoke/trade.cy.js:5]: We can remove the `beforeEach` if no longer needed
```suggestion
```

File: cypress/e2e/smoke/resources.cy.js (hunk 5 above)
Comment (markodonnell-deriv) [reply cypress/e2e/smoke/resources.cy.js]: Agreed, done

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**